wire-ios/Wire-iOS/Sources/UserInterface/Conversation/ConversationViewController.swift

```diff
@@ -151,7 +151,7 @@ final class ConversationViewController: UIViewController {
     override func viewDidLoad() {
         super.viewDidLoad()
 
-        if let session = session as? ZMUserSession {
+        if let session = userSession as? ZMUserSession {
```

Comment:
Can we avoid casting back to `ZMUserSession`?

Reply:
That's also done!